contrib/simdcomp-cmake/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+option (ENABLE_SIMDCOMP "Enable simdcomp" ${ENABLE_LIBRARIES})
+
+if (NOT ENABLE_SIMDCOMP)
+    message(STATUS "Not using simdcomp")
+    return()
+endif()
+
+set (LIBRARY_DIR "${ClickHouse_SOURCE_DIR}/contrib/simdcomp")
+set (SRCS
+    "${LIBRARY_DIR}/src/avx512bitpacking.c"
+    "${LIBRARY_DIR}/src/avxbitpacking.c"
+    "${LIBRARY_DIR}/src/simdbitpacking.c"
+    "${LIBRARY_DIR}/src/simdcomputil.c"
+    "${LIBRARY_DIR}/src/simdfor.c"
+    "${LIBRARY_DIR}/src/simdintegratedbitpacking.c"
+    "${LIBRARY_DIR}/src/simdpackedsearch.c"
+    "${LIBRARY_DIR}/src/simdpackedselect.c"
+)
+
+add_library (_simdcomp ${SRCS})
+
+target_include_directories (_simdcomp SYSTEM PUBLIC
+    "${LIBRARY_DIR}/include"
+    "${LIBRARY_DIR}/src"
+)
+
+if (OMIT_HEAVY_DEBUG_SYMBOLS)
```

Comment:
I wouldn't expect a tiny C library to produce heavy debug symbols. I think it's safe to omit l. 27-29.